### ISSUE TITLE
[Keyboard Navigation-AskHR Virtual Assistant]: Keyboard focus indicator is not visible when focus moves to the 'Send' button present in the page, while navigating using Keyboard Tab key.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#4297]
### PWD Impact:
Users who rely on keyboard for navigation are losing their track of focus as, Keyboard focus indicator is not visible when focus moves to the 'Send' button present in the page, while navigating using Keyboard Tab key. Users will not know which interactive control has focus as the focus indicator is not visible.

### Test Environment:
OS: Windows 10 version 21H1 (OS Build 19043.1110)
Browser: New Edge Version 92.0.902.62 (Official build) (64-bit)/Chrome Version 92.0.4515.107 (Official Build) (64-bit)
URL: [AskHR Virtual Assistant (azureedge.net)](https://askhrvauat.azureedge.net/?botorigin=hrweb) 

Repro Steps:
Step 1: Open the above URL in the Chrome/Edge Browser using valid credentials.
Step 2: 'AskHR Virtual Assistant' Home page gets opened.
Step 3:  Now navigate through the page by using keyboard tab key.
Step 4 :Verify whether  Keyboard focus indicator is not visible when focus moves to the 'Send' button present in the page  or not.

### Actual Result:
Keyboard focus indicator is not visible when focus moves to the 'Send' button present in the page, while navigating using Keyboard Tab key.

### Expected Result:
Keyboard focus indicator should be visible when focus moves to the 'Send' button present in the page, while navigating using Keyboard Tab key.

**Note:** Focus indicator should be visible on 'Type your message' edit field as well.

**Ensure** that: On traversing to the controls in HC modes the focus indicator should be clearly visible. 

### MAS Reference:
[MAS 2.4.7 – Focus Visible.docx](https://microsoft.sharepoint.com/:w:/s/accessibility/ERrM1V5kR81Aj1PIEHug1aIBa9x1n_rgutF9fnn8xmLcnA?e=bYduen)

### WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/focus-visible

**Comment by KattaBhargav** [comment]
Bug copied from CSEO team Bug id:-[BUG 7994935](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/7994935)

**Comment by axelsrz** [comment]
@compulim could you take a look at this one?

**Comment by cwhitten** [comment]
This is not an issue with Web Chat. It is an issue with the implementation of Web Chat. Closing
